frontend/javascripts/oxalis/view/components/editable_text_label.tsx

```diff
@@ -8,11 +8,12 @@ import Toast from "libs/toast";
 type Rule = {
   message?: string;
   type?: string;
+  validator?: (arg0: number | string) => boolean;
```

Comment:
Why is this `number | string`? As far as I can see, validator is only called with `this.state.value` which is `string`.